packages/server/graphql/mutations/helpers/notifySlack.ts

```diff
@@ -131,25 +203,25 @@ const upsertSlackMessage = async (
     const {channel} = convoInfo
     const {latest} = channel
     if (latest) {
-      const {ts, bot_profile} = latest
-      const {name} = bot_profile
-      if (name === 'Parabol') {
```

Comment:
Removed the check for the `Parabol` bot name as `latest` only exists in the DM and we handle these checks server side. Small advantages being it's slightly cleaner and we don't need to change the name to `Parabol-staging` in development